include/nccl_ofi_rdma.h

```diff
@@ -71,6 +71,16 @@ static_assert(MAX_NUM_RAILS <= UINT16_MAX);
  */
 #define NCCL_OFI_RDMA_SEQ_BITS     (10)
 
+/*
+ * @brief	Message sequence number bitmask for immediate data
+ */
+#define MSG_SEQ_NUM_MASK (((uint64_t)1 << NCCL_OFI_RDMA_SEQ_BITS) - 1)
+
+/*
+ * @brief	Number of bits used for number of segments value
+ */
+#define NUM_NUM_SEG_BITS ((uint64_t)4)
```

Comment:
Please document maximum segments supported is 15 or whatever is the reason for choosing 4.